ipfshttpclient/client/__init__.py

```diff
@@ -254,19 +260,55 @@ def add_bytes(self, data: bytes, **kwargs):
 		----------
 		data
 			Content to be added as a file
+		trickle
+			Use trickle-dag format (optimized for streaming) when generating
+			the dag; see `the old FAQ <https://github.com/ipfs/faq/issues/218>`_
+			for more information
+		only_hash
+			Only chunk and hash, but do not write to disk
+		chunker
+			The chunking algorithm to use
+		pin
+			Pin this object when adding
+		raw_leaves
+			Use raw blocks for leaf nodes. (experimental). (Default: ``True``
```

Comment:
What's the rationale for enabling the experimental `raw blocks` feature when `nocopy` is enabled? Since it's experimental, can `raw blocks` be left disabled by default even when `nocopy` is being used?

Note - I'm not yet suggesting the behavior must change, but rather seeing to understand the choice first.

Reply:
There is no rationale, I'm working by copy/pasting here. I was just trying to scratch an itch (e.g. why `add()` has `only_hash=True` but `add_str()` did not.

Reply:
Sounds like we need @ntninja to provide some more background on this.

Reply:
`raw_leaves` is a prerequisite for `nocopy` on the daemon side because of how no-copy blocks are stored there.

Also I already asked @ftyers to remove the `nocopy` argument because it does not make sense for content not added from local files or URLs. So all that is really needed here is:

  1. Remove any mentioning of `nocopy` in the docs (the entire *Default:* part can go)
  2. Make the argument default to `False` instead of `None`
  3. Change `"raw-leaves": raw_leaves if raw_leaves is not None else nocopy,` to just `"raw-leaves": raw_leaves,` below (that should also fix the test failure)

Reply:
Is `raw blocks` still an experimental feature? (ignoring that the entire IPFS is v0.x lol)

Is some adjustment needed to both this new docstring and the existing docstrings mentioning raw blocks / raw leaves?